pkg/api/openshiftcluster.go

```diff
@@ -360,6 +360,10 @@ type NetworkProfile struct {
 	LoadBalancerProfile        *LoadBalancerProfile `json:"loadBalancerProfile,omitempty"`
 }
 
+const (
+	MachineCIDR = "100.64.0.0/16"
```

Comment:
Why are we naming this Const as Machine CIDR? 

This CIDR range is used by OVN-kubernetes internally and as per the OVN code it is called as a Join CIDR. Using MachineCIDR can create a confusion with the actual ARO's Machine CIDR I guess.?

https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/config/config.go#L138

Reply:
Done